### DISCUSSION TITLE
Feedback for Math Class

### DISCUSSION THREAD

**Discussion by jcollard** [CaptainCoderOrg/IntroToCSharpSite#136] [Page Feedback]
Feedback discussion for Type Conversions: https://csharp-develop.captaincoder.org/lessons/variables/math-class

**Comment by cloudthepain** [comment]
At the beginning, I'd add a section for a quick refresher going over the concepts already covered really quickly.

"We've seen that we can add, multiply and divide already..." that way everything is together for them to see.

**Comment by cloudthepain** [comment]
For Math.Min / Max I would tell them something like, the first number is going to be the lowest value that the number can go while the second number is the highest the value can go.

**Comment by cloudthepain** [comment]
"The Math.Clamp method expects 3 arguments: value, min, max and evaluates to value if it is between min and max otherwise, it "clamps" to the max or min of the specified range." 

   -This section is a little hard to read,
